src/SFA.DAS.AdminService.Web/Startup.cs

```diff
@@ -83,6 +84,12 @@ public void ConfigureServices(IServiceCollection services)
                 options.SupportedCultures = new List<CultureInfo> { new CultureInfo("en-GB") };
                 options.RequestCultureProviders.Clear();
             });
+
+            services.Configure<RazorViewEngineOptions>(o =>
+            {
+                o.ViewLocationFormats.Add("/Views/RoatpAssessor/{1}/{0}" + RazorViewEngine.ViewExtension);
```

Comment:
Could you place a comment to say why you're doing stuff to the View Location schemes... this could have unwanted side affects

Reply:
The thinking is it means in my views I can use convention to return the required view and keep the controllers terse. So I can go:
`Return View();`
instead of
`Return View("~/Views/RoatpAssessor/Gateway/Index.cshtml");`

The location is added to the end of this list so its the last place to to be checked. All other views will have been resolved by then.

Yep I'm specifying execution root.  I could have added tilde so its more portable.

I'm not wedded to this approach so I'll remove as it has caused confusion. 👍